### DISCUSSION TITLE
Не скачиваются приложения из Googleplay

### DISCUSSION THREAD

**Discussion by StressOzz** [bol-van/zapret#850] [Q&A]
Здравствуйте.
Установил Zapret на OpenWrt. Всё работает. Спасибо !
Но на Андроид устройствах не скачиваются приложения из Googleplay.
Добавление доменов в исключение не помогает.

**Comment by numbereleven-a** [comment]
какая стратегия и мод фильтр?

**Comment by StressOzz** [comment]
> какая стратегия и мод фильтр?

--filter-tcp=443
--hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
--dpi-desync=fake,split2
--dpi-desync-repeats=11
--dpi-desync-fooling=md5sig
--dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin
--new ^
--filter-tcp=80,443
--dpi-desync=fake,disorder2
--dpi-desync-repeats=6
--dpi-desync-autottl=2
--dpi-desync-fooling=md5sig
--new ^
--filter-udp=443
--hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
--dpi-desync=fake
--dpi-desync-repeats=11
--dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin
--new ^
--filter-udp=443
--dpi-desync=fake
--dpi-desync-repeats=11

**Comment by numbereleven-a** [comment]
то есть без запрета они скачиваются, а с запретом они не скачиваются?

**Comment by StressOzz** [comment]
> то есть без запрета они скачиваются, а с запретом они не скачиваются?

да, с включённым Запретом Googleplay открывается и всё там можно посмотреть, но приложения не скачиваются - вечная надпись "подождите"....

**Comment by numbereleven-a** [comment]
zapret-hosts-google.txt
что там

**Comment by StressOzz** [comment]
> zapret-hosts-google.txt что там

не чего не меня, ставил как есть...

googlevideo.com
youtubei.googleapis.com
ytimg.com
yt3.ggpht.com
yt4.ggpht.com
youtube.com
youtubeembeddedplayer.googleapis.com
ytimg.l.google.com
jnn-pa.googleapis.com
youtube-nocookie.com
youtube-ui.l.google.com
yt-video-upload.l.google.com
wide-youtube.l.google.com

**Comment by numbereleven-a** [comment]
Попробуй в исключение (zapret-hosts-user-exclude.txt) добавь play.google.com либо куда оно там ломится
Попробуй подбери страту без ttl либо без autottl, а поискать цифровое значение

**Comment by StressOzz** [comment]
> play.google.com

play.google.com добавлял и туда тоже...
Спасибо за совет, буду копать )

**Comment by numbereleven-a** [comment]
> Googleplay

на 4пда такой список помог человеку
: <details><summary>spoiler</summary>![image](https://github.com/user-attachments/assets/b907de9e-5347-4c3c-8f39-ed957adf2d35)#+html: </details>

**Reply by playrusboy** [reply]
Отдуши , мне помогло

**Reply by chipodrillo** [reply]
Тоже не качает с гуглплэй.. куда вы эти домены добавляли? в отдельный лист к сайтам или к списку для ютуба?

**Reply by playrusboy** [reply]
Host lists 
User excluded hostname entries

**Comment by Noflat258** [comment]
Вот это все в конфиге:
```
--filter-tcp=80,443
--dpi-desync=fake,disorder2
--dpi-desync-repeats=6
--dpi-desync-autottl=2
--dpi-desync-fooling=md5sig
```
применяется ко всему. hostlists ссылки тут нет. Оно и ломает.

**Comment by StressOzz** [comment]
> Вот это все в конфиге:
> 
> ```
> --filter-tcp=80,443
> --dpi-desync=fake,disorder2
> --dpi-desync-repeats=6
> --dpi-desync-autottl=2
> --dpi-desync-fooling=md5sig
> ```
> 
> применяется ко всему. hostlists ссылки тут нет. Оно и ломает.

Здравствуйте.
Только это вставить или как ?
Опишите пожалуйста по подробней )

**Comment by Noflat258** [comment]
Я думаю вот так:

```
--filter-tcp=443
--hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
--dpi-desync=fake,split2
--dpi-desync-repeats=11
--dpi-desync-fooling=md5sig
--dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin
--new
--filter-tcp=80
--hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
--dpi-desync=fake,disorder2
--dpi-desync-repeats=6
--dpi-desync-autottl=2
--dpi-desync-fooling=md5sig
--dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin
--new
--filter-udp=443
--hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
--dpi-desync=fake
--dpi-desync-repeats=11
--dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin
```

**Comment by StressOzz** [comment]
> Я думаю вот так:
> 
> ```
> --filter-tcp=443
> --hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
> --dpi-desync=fake,split2
> --dpi-desync-repeats=11
> --dpi-desync-fooling=md5sig
> --dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin
> --new
> --filter-tcp=80
> --hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
> --dpi-desync=fake,disorder2
> --dpi-desync-repeats=6
> --dpi-desync-autottl=2
> --dpi-desync-fooling=md5sig
> --dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin
> --new
> --filter-udp=443
> --hostlist=/opt/zapret/ipset/zapret-hosts-google.txt
> --dpi-desync=fake
> --dpi-desync-repeats=11
> --dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin
> ```

Да, спасибо. Googleplay работает. Но "запрещённые" сайты не открывает (
Как их добавить и куда ?

**Comment by Noflat258** [comment]
сюда, в этот файл, просто в конец списка добавить новые домены:
nano /opt/zapret/ipset/zapret-hosts-google.txt
Пищите какие. Домен как правило не один.
